modules/basics/src/main/java/com/opengamma/strata/basics/schedule/PeriodicSchedule.java

```diff
@@ -109,9 +106,8 @@
  * When the unadjusted schedule has been determined, the appropriate business day adjustment
  * is applied to create a parallel schedule of "adjusted" dates.
  */
-@BeanDefinition
-public final class PeriodicSchedule
-    implements ImmutableBean, Serializable {
+@BeanDefinition(style = "minimal")
```

Comment:
Switched this over to minimal to stop us coming up against our checkstyle 2000 line rule.